packages/sqs-receiver-service/src/receiver.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+import db from 'debug'
+import Bottleneck from 'bottleneck'
+import environment from './environment.js'
+import readQueue from './read-queue.js'
+import processMessage from './process-message.js'
+import deleteMessages from './delete-messages.js'
+import showQueueStatistics from './show-queue-statistics.js'
+
+/**
+ * This is the process for a single SQS queue
+ */
+const debug = db('receiver')
+
+// Validate the environment and return a standard object
+const { env } = environment(process.env, process.env.RECEIVER_PREFIX)
+
+console.log(`Running receiver process:${JSON.stringify(env, null, 4)}`)
+
+const limiter = new Bottleneck({
```

Comment:
Without redis this isn't going to work as expected when operating in a cluster

Reply:
I understand that